src/components/node/stanford-page/home-page/home-page-banner.tsx

```diff
@@ -11,14 +11,15 @@ const HomePageBanner = async () => {
 
   // Trim all the fat.
   const trimmedLibraries: TrimmedLibrary[] = libraries.map(library => (
-    {
-      id: library.id,
-      title: library.title,
-      suLibraryHours: library.suLibraryHours,
-      suLibraryContactImg: library.suLibraryContactImg as MediaImage,
-      suLibraryBanner: library.suLibraryBanner as MediaImage
-    }
-  ))
+      {
+        id: library.id,
+        title: library.title,
+        suLibraryHours: library.suLibraryHours,
+        suLibraryContactImg: library.suLibraryContactImg as MediaImage,
+        suLibraryBanner: library.suLibraryBanner as MediaImage,
+        map: library.suLibraryMapLink?.url ?? undefined
```

Comment:
You don't need the `?? undefined`

Reply:
It threw a typescript error without it.  Because of the `Maybe` I think.

Reply:
Yeah. If you use `Maybe<string>` it should fix that.